--- /home/Gnana/.config/hypr/hyprland.lua	2026-09-19 09:22:44.602777574 +0530
+++ .config/hypr/hyprland.lua	2026-09-17 13:14:50.701938071 +0530
@@ -259,7 +259,7 @@
         },
         layout = "dwindle",
         -- Please see https://wiki.hyprland.org/Configuring/Tearing/ before you turn this on
-        allow_tearing = false,
+        allow_tearing = true,
     },
     decoration = {
         -- See https://wiki.hyprland.org/Configuring/Variables/ for more
